src/coreclr/src/hosts/unixcorerun/CMakeLists.txt

```diff
@@ -1,33 +1,20 @@
 project(unixcorerun)
 
-include_directories(../unixcoreruncommon)
+include_directories("${CLR_SRC_NATIVE_DIR}/common")
+include(configure.cmake)
 
-set(CORERUN_SOURCES
-    corerun.cpp
-)
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
 _add_executable(corerun
-    ${CORERUN_SOURCES}
+    corerun.cpp
+    coreruncommon.cpp
```

Comment:
Maybe we could even merge the two files now.

Reply:
Yup, deleted the header as well.

Reply:
In addition to unixcorerun, should we continue to compile corerun on (non-Linux) Unix?
https://github.com/dotnet/runtime/blob/8861c419d23a442490fee524c67ce02b10dbb154/src/coreclr/src/hosts/corerun/CMakeLists.txt#L10-L17

Reply:
This is obsolete, it was never being compiled on Unix in .NET Core. It seems like I have forgotten it there from the initial phases of .NET Core porting to Unix. Thank you for noticing it, it would be great to remove it.

Reply:
Sure, it is controlled by src/coreclr/src/hosts/CMakeLists.txt (where this directory is not included). I will clean up the cmake lists file.